### ISSUE TITLE
Boost cache issue on www.hotosm.org

### ISSUE THREAD

**Issue by bgirardot** [hotosm/hotosm-website#39]
I know this is a duplicate of a closed issue, but it still seems to be happening. I don't know how much of a concern it is since I don't know how much traffic that domain gets as opposed to the hot.openstreetmap.org domain but I thought I would make an issue for it just in case there is something we can do about it.

www.hotosm.org is days behind hot.openstreetmap.org with a boost cache comment that reads:

```
<!-- Page cached by Boost @ 2015-02-24 12:32:05, expires @ 2015-02-24 13:32:05, lifetime 1 hour -->
```

Not quite 3 days old.

**Comment by xamanu** [comment]
This still happens, and with the latest Nepal activation and the attention on the website, this is critical. As people might not find the expected information when visiting the website.

Somehow also related to #6

**Comment by xamanu** [comment]
Cool, the domains are unified and we get only one content, no different anymore.

I had a further look on the caching and I realized that the cron job (no idea where it is getting called from) was already relatively old (almost 2 hours) and the caching was put to a maximum lifetime of 1 hour, which is quite crucial, because the frontpage is a view and not a direct update is happening to it (like it would be on a node) and therefore it is not getting updated, or very late. Because of very few cron jobs and a too high caching time.

I assured to have now a cron job running every five minutes and the maximum caching lifetime for the html boost cache I set down to 10 minutes.

So in theory, in 5 to 15 minutes after publishing the content should be online for not-logged-in-users.

I leave this open a couple of days. Till people can confirm that they are happy how it is working now.

Cheers
